lib/rucio/web/rest/flaskapi/v1/rules.py

```diff
@@ -827,21 +828,21 @@ def blueprint() -> AuthenticatedBlueprint:
     bp = AuthenticatedBlueprint('rules', __name__, url_prefix='/rules')
 
     rule_view = Rule.as_view('rule')
-    bp.add_url_rule('/<rule_id>', view_func=rule_view, methods=['get', 'put', 'delete'])
+    bp.add_url_rule('/<rule_id>', view_func=rule_view, methods=[HTTPMethod.GET.value, HTTPMethod.PUT.value, HTTPMethod.DELETE.value])
     all_rule_view = AllRule.as_view('all_rule')
-    bp.add_url_rule('/', view_func=all_rule_view, methods=['get', 'post'])
+    bp.add_url_rule('/', view_func=all_rule_view, methods=[HTTPMethod.GET.value, HTTPMethod.POST.value])
```

Comment:
Small question - do we expect any possible issue that might arise by substituting these lowercase methods with their uppercase equivalents?